src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -291,4 +304,15 @@ class ExternalIssueActions extends AsyncComponent {
   }
 }
 
+const IssueTitle = styled('div')`
+  font-size: 1.1em;
+  font-weight: 600;
+  ${overflowEllipsis};
+`;
+
+const IssueDescription = styled('div')`
+  margin-top: ${space(1)};
+  ${overflowEllipsis};
+`;
```

Comment:
what about allowing it to display a few lines and then just overflow hidden, with a white fade at the bottom?

Reply:
Not a bad idea! A little tricky implimentation-wise tho because we'd need to test the height of the text which depends on the combination of characters, otherwise the white fade would be there even if the text wasn't long enough. 

It can also be tricky to get the design right for that effect, because the gradient needs to be "just right" in combination with the text size and line-height for a nice looking fade. Perhaps for v2.